<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Clicked on Creat Account Button<br>]]>
    </line>
  </reporter-output>
  <suite started-at="2024-08-11T17:44:47 BST" name="Default suite" finished-at="2024-08-11T17:44:48 BST" duration-ms="896">
    <groups>
    </groups>
    <test started-at="2024-08-11T17:44:47 BST" name="Default test" finished-at="2024-08-11T17:44:48 BST" duration-ms="896">
      <class name="com.testProject.testcases.CreateUserAccountTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.testProject.testcases.CreateUserAccountTest@3c435123]" started-at="2024-08-11T17:44:43 BST" name="setUp" finished-at="2024-08-11T17:44:47 BST" duration-ms="4791" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="createUserTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.testProject.testcases.CreateUserAccountTest@3c435123]" started-at="2024-08-11T17:44:47 BST" name="createUserTest" data-provider="getData" finished-at="2024-08-11T17:44:48 BST" duration-ms="834" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ahmed]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[khan]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ahmed@testmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Datnight123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Datnight123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testProject.testcases.CreateUserAccountTest.createUserTest(CreateUserAccountTest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Clicked on Creat Account Button<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- createUserTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testProject.testcases.CreateUserAccountTest@3c435123]" started-at="2024-08-11T17:44:48 BST" name="tearDown" finished-at="2024-08-11T17:44:48 BST" duration-ms="149" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testProject.testcases.CreateUserAccountTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
